modules/insticatorBidAdapter.js

```diff
@@ -178,6 +185,48 @@ function buildImpression(bidRequest) {
     imp.video = buildVideo(bidRequest);
   }
 
+  if (isFn(bidRequest.getFloor)) {
+    let moduleBidFloor;
+
+    const mediaType = deepAccess(bidRequest, 'mediaTypes.banner') ? 'banner' : deepAccess(bidRequest, 'mediaTypes.video') ? 'video' : undefined;
+
+    let _mediaType = mediaType;
+    let _size = '*';
```

Comment:
what is the logic here for size? and how's it related to bid floor update?

Reply:
We have a common function `getFloor` which is module specfic function used to get the floors. 
The getFloor function requires mediatype and size as required param in order to fetch the floor.

Here is the logic for banner:
``` banner: {
format: [[300,250], [350,200]]
w: 300,
h: 250,
}
```

The above is example for banner, in which we can either have format or w&h. 
Therefore for banner we fetch size in a specific way.

Logic for video mediatype:

- We extract the w & h directly from the imp object.

@rohanInsticator

Reply:
I know we can have size in 2 fields but the floor is still at imp level, hence I asked

Reply:
ohhh, for that I was following these guideline
https://docs.prebid.org/dev-docs/modules/floors.html#bid-adapter-interface

If floor in the modules are size specific then we will fetch according to it, otherwise we fetch floor for all sizes by using `*`.